Formula/q/qcachegrind.rb

```diff
@@ -26,24 +23,19 @@ class Qcachegrind < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "e7a3dd4ae5be833f996b409c8db1908baff72e5f053a118bb7b8f496dc829b5a"
   end
 
-  head do
-    url "https://invent.kde.org/sdk/kcachegrind.git", branch: "master"
-    depends_on "qt"
-  end
-
   depends_on "graphviz"
+  depends_on "qt"
 
   fails_with gcc: "5"
 
   def install
     args = []
     if OS.mac?
-      # TODO: when using qt 6, modify the spec
       spec = (ENV.compiler == :clang) ? "macx-clang" : "macx-g++"
       args = %W[-config release -spec #{spec}]
```

Comment:
Do we not want to pass `-config release` on Linux?

Reply:
Done